### PR TITLE
Update Packages - warning auth_providers

### PR DESCRIPTION
**Pull request by ArveVM**
## Proposed change

Adding reference to limitations of loading auth_provider configuration in Packages

## Type of change

- [] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/configuration/packages.markdown [hunk 1]
```diff
@@ -110,3 +110,10 @@ It is possible to [customize entities](/docs/configuration/customizing-devices/)
 homeassistant:
   customize:
 ```
+
+
+<div class='note warning'>
+
+If you are moving configuration to packages, auth_providers must stay within ‘configuration.yaml’. See [Issue 16441](https://github.com/home-assistant/core/issues/16441) and/ general documentation for [Authentication Providers](https://www.home-assistant.io/docs/authentication/providers/#configuring-auth-providers)
+
+This is because Home Assistant processes the `auth_provider` during the `core` section loading, which is earlier than the `packages` processing.
```
Comment (c0ffeeca7) [inline source/_docs/configuration/packages.markdown]:
```suggestion
This is because Home Assistant processes the `auth_provider` during the `core` section loading, which is done before the `packages` are processed.
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

**Review by ArveVM — COMMENTED** [review]
great improvements:)

File: source/_docs/configuration/packages.markdown [hunk 2]
```diff
@@ -110,3 +110,10 @@ It is possible to [customize entities](/docs/configuration/customizing-devices/)
 homeassistant:
   customize:
 ```
+
+
+<div class='note warning'>
+
+If you are moving configuration to packages, auth_providers must stay within ‘configuration.yaml’. See [Issue 16441](https://github.com/home-assistant/core/issues/16441) and/ general documentation for [Authentication Providers](https://www.home-assistant.io/docs/authentication/providers/#configuring-auth-providers)
```
Comment (c0ffeeca7) [inline source/_docs/configuration/packages.markdown]:
```suggestion
If you are moving configuration to packages, `auth_providers` must stay within ‘configuration.yaml’. See [Issue 16441](https://github.com/home-assistant/core/issues/16441) and the general documentation for [Authentication Providers](https://www.home-assistant.io/docs/authentication/providers/#configuring-auth-providers).
```

**Review by c0ffeeca7 — DISMISSED** [review]
I missed that one before.

File: source/_docs/configuration/packages.markdown [hunk 3]
```diff
@@ -110,3 +110,10 @@ It is possible to [customize entities](/docs/configuration/customizing-devices/)
 homeassistant:
   customize:
 ```
+
+
+<div class='note warning'>
+
+If you are moving configuration to packages, `auth_providers` must stay within ‘configuration.yaml’. See [Issue 16441](https://github.com/home-assistant/core/issues/16441) and the general documentation for [Authentication Providers](https://www.home-assistant.io/docs/authentication/providers/#configuring-auth-providers).
```
Comment (frenck) [inline source/_docs/configuration/packages.markdown]: We should not send end-users into GitHub issues for information. If information from that linked issue is relevant, please add it to the docs and remove the issue link.

Comment (frenck) [inline source/_docs/configuration/packages.markdown]: The link used to other parts of our documentation must be relative to the root of our website (and not a full link).
`/docs/authentication/providers/#configuring-auth-providers`

File: source/_docs/configuration/packages.markdown [hunk 4]
```diff
@@ -110,3 +110,10 @@ It is possible to [customize entities](/docs/configuration/customizing-devices/)
...
+
+This is because Home Assistant processes the `auth_provider` during the `core` section loading, which is done before the `packages` are processed.
```
Comment (frenck) [inline source/_docs/configuration/packages.markdown]: I'm not sure if users realize what "`core` section loading" is. Maye reword this as:

> Home Assistant processes the authentication provided early in the start-up process, even before packages are processed.

File: source/_docs/configuration/packages.markdown [hunk 5]
```diff
@@ -110,3 +110,10 @@ It is possible to [customize entities](/docs/configuration/customizing-devices/)
 homeassistant:
   customize:
 ```
+
+
+<div class='note warning'>
```
Comment (frenck) [inline source/_docs/configuration/packages.markdown:115]: This `<div>` seems unclosed?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_docs/configuration/packages.markdown (hunk 5 above)
Comment (ArveVM) [reply source/_docs/configuration/packages.markdown:115]: added new line 119

File: source/_docs/configuration/packages.markdown (hunk 4 above)
Comment (ArveVM) [reply source/_docs/configuration/packages.markdown]: me neither ;),,, just copied text from auth-provider-docs,,,, 
Keeping the link/reference to it should be sufficient ?

File: source/_docs/configuration/packages.markdown (hunk 3 above)
Comment (ArveVM) [reply source/_docs/configuration/packages.markdown]: corrected

Comment (ArveVM) [reply source/_docs/configuration/packages.markdown]: removed,, link to auth-providers-doc is enough

File: source/_docs/configuration/packages.markdown [hunk 6]
```diff
@@ -110,3 +110,10 @@ It is possible to [customize entities](/docs/configuration/customizing-devices/)
...
+If you are moving configuration to packages, `auth_providers` must stay within ‘configuration.yaml’. See the general documentation for [Authentication Providers](/docs/authentication/providers/#configuring-auth-providers).
+
+This is because Home Assistant processes the authentication provided early in the start-up process, even before packages are processed.
```
Comment (frenck) [inline source/_docs/configuration/packages.markdown:119]: Adds new-lines to ensure parsers can switch between HTML and Markdown context.

```suggestion

If you are moving configuration to packages, `auth_providers` must stay within ‘configuration.yaml’. See the general documentation for [Authentication Providers](/docs/authentication/providers/#configuring-auth-providers).

This is because Home Assistant processes the authentication provided early in the start-up process, even before packages are processed.

```

**Review by frenck — APPROVED** [review]
Thanks, @ArveVM 👍 

../Frenck
